web/src/constants/TestSpecs.constants.ts

```diff
@@ -0,0 +1,45 @@
+import {IValues} from 'components/TestSpecForm/TestSpecForm';
+
+export type TSnippet = Required<IValues>;
+
+export const HTTP_SPANS_STATUS_CODE: TSnippet = {
```

Comment:
Here's where we can continue adding snippets, you can add it manually using the UI and then copy the details in this file @kdhamric